tutorials/tour/_posts/2017-02-13-annotations.md

```diff
@@ -11,79 +11,51 @@ next-page: default-parameter-values
 previous-page: automatic-closures
 ---
 
-Annotations associate meta-information with definitions.
+Annotations associate meta-information with definitions. For example, the annotation `@deprecated` before a method causes the compiler to print a warning if the method is used.
+```tut
+@deprecated
+def hello = "hola"
 
-A simple annotation clause has the form `@C` or `@C(a1, .., an)`. Here, `C` is a constructor of a class `C`, which must conform to the class `scala.Annotation`. All given constructor arguments `a1, .., an` must be constant expressions (i.e., expressions on numeral literals, strings, class literals, Java enumerations and one-dimensional arrays of them).
+hello
+```
+This will compile but the compiler will print a warning: "there was one deprecation warning
```

Comment:
I think "print a warning" or "print a deprecation warning" is sufficient here. keeping the focus on annotations